docs/apps/components/button.md

```diff
@@ -0,0 +1,83 @@
+# Button
+
+## Description
+
+A button can be used to trigger an event when it's clicked.
+
+Buttons are typically used to submit forms, navigate between pages, or perform other actions.
+
+<img src= "/apps/components/img/button.png">
+
+## Use Case
+
+Some Use Cases are listed as follows:
```

Comment:
The word casing in this entire Use Case section is not consistent. I understand you want the words to stand out, but you could make the works bold, or underline or italics or something instead of changing the casing mid-sentence when the word is not a Noun. e.g 'Use Cases' that follows the word Some, should be in lowercase.

Reply:
done